cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -88,19 +179,44 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
         query("SHOW USERS", assertNodesShown("User", column = "user")) {
           resultTable()
         }
+        p("It is possible to assign multiple roles to multiple users in one command.")
+        query("GRANT ROLES role1, role2 TO user1,user2,user3", ResultAssertions( r => {
```

Comment:
You have whitespace in the first list, not not the second. I think adding whitespace to the second list will look better.